nuxt.config.ts

```diff
@@ -1,5 +1,6 @@
 import nodePolyfills from 'rollup-plugin-polyfill-node'
 import { copyAssets } from '@lukso/web-components/tools/copy-assets'
+// @ts-ignore
```

Comment:
why is ts complaining about?

Reply:
We have again some TS issues in web components library after recent package update, basically the definition files are not generated properly. This is temp fix, I'll remove ignores once we have things working back again.